searchlib/src/vespa/searchlib/attribute/postinglistsearchcontext.h

```diff
@@ -151,8 +149,8 @@ class PostingListFoldedSearchContextT : public PostingListSearchContextT<DataT>
     using Parent::singleHits;
 
     PostingListFoldedSearchContextT(const Dictionary &dictionary, uint32_t docIdLimit, uint64_t numValues,
-                                    bool hasWeight, const PostingList &postingList, const IEnumStore &esb,
-                                    uint32_t minBvCocFreq, bool useBitVector, const ISearchContext &baseSearchCtx);
+                                    bool hasWeight, const PostingList &postingList, uint32_t minBvCocFreq,
+                                    bool useBitVector, const ISearchContext &baseSearchCtx);
```

Comment:
No need to keep old typos, consider using `minBvDocFreq`

Reply:
Fixed